auth-web/src/services/business.services.ts

```diff
@@ -93,4 +94,111 @@ export default class BusinessService {
+    if (!reviewId) {
+      throw new Error('Invalid parameters')
+    }
+
+    // *** TODO: once the API is ready, replace this mocked data with the Axios call below
+    return new Promise(resolve => resolve({
+      review: {},
+      results: [],
+      filing: {
+        continuationIn: {
+          authorization: {
+            date: '2024-07-01',
+            files: [
+              {
+                fileKey: '0071dbd6-6095-46f6-b5e4-cc859b0ebf27.pdf',
+                fileName: 'My Authorization Document.pdf'
+              },
+              {
+                fileKey: 'xxx.pdf',
+                fileName: 'Invalid File.pdf'
+              }
+            ]
+          },
+          business: {
+            foundingDate: '2001-05-03T07:00:00.000+00:00',
+            identifier: 'A0054444',
+            legalName: 'FIRST AWIQ SHOPPING CENTRES BC LIMITED'
+          },
+          foreignJurisdiction: {
+            affidavitFileKey: '007bd7bd-d421-49a9-9925-03ce561d044f.pdf',
+            affidavitFileName: 'My Director Affidavit.pdf',
+            country: 'CA',
+            identifier: 'AB-5444',
+            incorporationDate: '2001-04-02',
+            legalName: 'FIRST AWIQ SHOPPING CENTRES ALBERTA UNLIMITED',
+            region: 'AB',
+            taxId: '123456789'
+          },
+          mode: 'EXPRO',
+          nameRequest: {
+            legalType: 'CUL'
+          }
+        }
+      }
+    }))
+
+    // const url = `${ConfigHelper.getLegalAPIV2Url()}/continuation-reviews/${reviewId}`
+    // return axios.get(url)
+  }
+
+  /**
+   * Downloads a Minio document from Legal API and prompts browser to open/save it.
+   * @param documentKey the document key
+   * @param documentName the document filename
+   * @returns a promise to return the axios response or the error response
+   * @see CommonUtils.fileDownload() for a similar method
+   */
+  static async downloadDocument (documentKey: string, documentName: string): Promise<AxiosResponse> {
+    // safety checks
+    if (!documentKey || !documentName) {
+      throw new Error('Invalid parameters')
+    }
+
+    const url = `${ConfigHelper.getLegalAPIV2Url()}/documents/${documentKey}`
+    const config = {
+      headers: { 'Accept': 'application/pdf' },
+      responseType: 'blob' as 'json'
+    }
+
+    return axios.get(url, config).then(response => {
+      if (!response) throw new Error('Null response')
+
+      /* solution below is from https://github.com/axios/axios/issues/1392 */
+
+      // it is necessary to create a new blob object with mime-type explicitly set
+      // otherwise only Chrome works like it should
+      const blob = new Blob([response.data], { type: 'application/pdf' })
+
+      // use Navigator.msSaveOrOpenBlob if available (possibly IE)
+      // warning: this is now deprecated
+      // ref: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/msSaveOrOpenBlob
+      if (window.navigator && window.navigator['msSaveOrOpenBlob']) {
```

Comment:
We don't support IE 11?

Reply:
I copied this from our other UIs.

I see that common-utils.ts has slightly different blob handling. Do yourecommend I update the code above?

https://github.com/bcgov/sbc-auth/blob/e2425634ee8cdf8c47b17d37bc773ed24a692592/auth-web/src/util/common-util.ts#L156